packages/esm-patient-registration-app/src/patient-registration/field/name/name-field.component.tsx

```diff
@@ -31,37 +33,51 @@ export const NameField = () => {
   const fieldConfigs = useFieldConfig('name');
 
   return (
-    <div>
-      <h4 className={styles.productiveHeading02Light}>{t('fullNameLabelText', 'Full Name')}</h4>
-      <div className={styles.grid}>
-        <Input
-          id="givenName"
-          name="givenName"
-          labelText={t('givenNameLabelText', 'Given Name')}
-          light
-          checkWarning={checkNumber}
-        />
-        <ExtensionSlot
-          extensionSlotName="capture-patient-photo-slot"
-          state={{ onCapturePhoto, initialState: currentPhoto }}
-        />
-        {fieldConfigs.displayMiddleName && (
+    <div className={desktop && styles.grid}>
+      <div>
+        <h4 className={styles.productiveHeading02Light}>{t('fullNameLabelText', 'Full Name')}</h4>
+        <div>
+          {!desktop && (
+            <div style={{ margin: '1rem 0' }}>
+              <ExtensionSlot
+                extensionSlotName="capture-patient-photo-slot"
+                state={{ onCapturePhoto, initialState: currentPhoto }}
+              />
+            </div>
+          )}
```

Comment:
The most important thing we need to fix about this PR is that we shouldn't have these duplicated sections that are in different places based on the screen size. Instead, the component should exist only once in code, and be reordered using the [Flex Order property](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Ordering_Flex_Items).

Reply:
Hi @brandones! So sorry for replying here so late, actually if we see the designs, I don't have any other option to go for other than this. The designs for the laptop and the table view is as follows: 

Desktop: 
![image](https://user-images.githubusercontent.com/51502471/135428451-46e4685c-fd7c-450b-9271-545d0316391d.png)

Tablet:
![image](https://user-images.githubusercontent.com/51502471/135428510-6fa07cdb-e500-4540-b2bf-ca7c5a276cbd.png)

I'll have a look at it, if I can get some good news!

Reply:
Not sure I understand. Why can't you use the Flex Order property to achieve the reordering of elements?

Reply:
Hey @brandones , I know it's very late, but I have worked upon and now it's working good.